packages/round-manager/.eslintrc.js

```diff
@@ -12,14 +12,11 @@ module.exports = {
   rules: {
     "react/no-unescaped-entities": ["error", { forbid: [">", "}"] }],
     "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-non-null-assertion": "off",
   },
   overrides: [
     {
       files: ["**/*.test.{ts,tsx}"],
       rules: {
-        "@typescript-eslint/no-empty-function": "off",
```

Comment:
i think we should leave the `no-empty-function: off` override for test files

Reply:
Added back the overrides for test files.

Reply:
Ah okie ! Makes sense to do that 🙌🏼